core/templates/pages/skill-editor-page/modal-templates/add-worked-example-modal.controller.ts

```diff
@@ -36,6 +36,12 @@ angular.module('oppia').controller('AddWorkedExampleModalController', [
     $scope.tmpWorkedExampleQuestionHtml = '';
     $scope.tmpWorkedExampleExplanationHtml = '';
 
+    $scope.isWorkedExampleQuestionHtmlLengthExceeded = () =>
+      $scope.tmpWorkedExampleQuestionHtml.length > 1000;
```

Comment:
Here and below, please use a named constant for the number.

Reply:
PTAL @kevintab95